### ISSUE TITLE
Dashboard tile: Accept other assets

### ISSUE THREAD

**Issue by irisli** [irisli/stellarterm#71]
Phrase it in a way for newbies to know that in order to accept other assets, they have to trust some anchors

**Comment by jedmccaleb** [comment]
Maybe this should just be the "deposit" tab?

**Comment by irisli** [comment]
Yeah, the term `Add trust` is really confusing. Maybe instead, we could use the word `Activate`. 

So initially, there is no `send` and `receive` button. Instead, it just says `Activate`. Once they press activate, trust will be added and then the send and receive buttons show.
